analysed-packages/dosfstools/version-4.2/dosfstools-4.2-SPDX2TV.spdx

```diff
@@ -0,0 +1,3339 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_dosfstools-4.2.tar.gz_1726571588.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of dosfstools 4.2.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-09-17T11:13:08Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: dosfstools-4.2
+PackageFileName: dosfstools-4.2.tar.gz
+SPDXID: SPDXRef-upload93
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 4ae57ec8e64dfd595571e4530dcbc2a1e86b3053
+PackageChecksum: SHA1: 14efc68c64987a9067b6a919ad2870007247686b
+PackageChecksum: SHA256: 64926eebf90092dca21b14259a5301b7b98e7b1943e8a201c7d726084809b527
+PackageChecksum: MD5: 49c8e457327dc61efab5b115a27b087a
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-GPL-3.0-only
```

Comment:
The license declared of the package is GPL-3.0-or-later because all  files which carry a reference to GPL-3.0 are licensed under GPL-3.0-or-later